docs/contributing/translations.md

```diff
@@ -68,8 +68,42 @@ _("Page {page_title} with status {status}".format(page_title=page.title, status=
 _(f"Page {page.title} with status {page.status_string}")
 ```
 
+For example (in template):
+  -   `some_tag` is a placeholder for any tag.
+  -   `arg_of_some_tag` is a placeholder for an argument of a tag.
+  -   `value_of_some_tag` is a placeholder for a value passed to a tag.
+  -   `filter` is a placeholder for a filter.
+
+```html
+{% extends "wagtailadmin/base.html" %}
+{% load i18n %}
+<!-- preliminary lines of code -->
+
+<!-- Do this to use the translate tag. -->
+{% translate "Any string of your choosing" %}
+
+<!-- Do this to use the blocktranslate tag. -->
+{% blocktranslate %}
+  A multiline translatable literal.
+{% endblocktranslate %}
+
+<!-- Do these to translate string literals passed to tags and filters. -->
+{% some_tag _("Any string of your choosing") %}
+{% some_tag arg_of_some_tag=_("Any string of your choosing") %}
+{% some_tag value_of_some_tag|filter=_("Any string of your choosing") %}
+
+<!-- A typical example of when to use translation of string literals is -->
+{% translate "example with literal" as var_name %}
+{% some_tag arg_of_some_tag=var_name %}
+<!-- If the variable is only ever used once, you could do this instead -->
+{% some_tag arg_of_some_tag=_("example with literal")%}
+```
+**Note**: In Wagtail code, you might see `trans` and `blocktrans` instead of `translate` and `blocktranslate`.
+This still works fine. `trans` and `blocktrans` were the tags earlier on in Django, but were replaced in [Django 3.1](https://docs.djangoproject.com/en/3.1/topics/i18n/translation/#translate-template-tag).
+
 ## Additional resources
 
 -   [](django:topics/i18n/translation)
 -   A screen-share [Wagtail Space US 2020 Lightning Talk](https://www.youtube.com/watch?v=sLI_AuOMUQw&t=17s) that walks through using Transifex step-by-step
 -   [Core development instructions for syncing Wagtail translations with Transifex](https://github.com/wagtail/wagtail/wiki/Managing-Wagtail-translations)
+-   [Django docs](https://docs.djangoproject.com/en/4.2/topics/i18n/translation/)
```

Comment:
```suggestion
-   [Django docs](django:topics/i18n/translation)
```

Using the topics system we can avoid links to specific versions of Django.

Reply:
Wow! Again, I didn't know about this part. Thank you!!